.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,102 @@
+      - name: Make all gradlew files executable
+        run: |
+          find project/backend -name "gradlew" -type f -exec chmod +x {} \;
+
+      - name: Update build.gradle files with JaCoCo fixes
+        run: |
+          for dir in project/backend/*/; do
+            if [ -f "$dir/build.gradle" ]; then
+              echo "Updating JaCoCo config in $dir/build.gradle"
+              # Backup the original file
+              cp "$dir/build.gradle" "$dir/build.gradle.bak"
+              
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)